### PR TITLE
docs(install): update deployment_guide to include AWS EKS Addon details

### PR DESCRIPTION
**Pull request by Chirag-CG**
**Purpose of PR?**:

- added section to describe KubeArmor EKS plugin add-on installation steps.

Fixes #

**Does this PR introduce a breaking change?**
No 

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [x] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`

### CODE REVIEW

File: getting-started/deployment_guide.md [hunk 1]
```diff
@@ -24,6 +24,50 @@ curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
 > [!NOTE] 
 > kArmor CLI provides a Developer Friendly way to interact with KubeArmor Telemetry. You can stream KubeArmor telemetry independently of kArmor CLI tool and integrate it with your chosen SIEM (Security Information and Event Management) solutions. [Here's a guide](https://github.com/kubearmor/kubearmor-relay-server/blob/main/README.md#streaming-kubearmor-telemetry-to-external-siem-tools) on how to achieve this integration. This guide assumes you have kArmor CLI to access KubeArmor Telemetry but you can view it on your SIEM tool once integrated.
 
+## KubeArmor EKS-plugin
```
Comment (daemon1024) [inline getting-started/deployment_guide.md]: Can we keep this heading entirely as a collapsible section under install. Next line to helm install.

File: getting-started/deployment_guide.md [hunk 2]
```diff
@@ -24,6 +24,50 @@ curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
 > [!NOTE] 
 > kArmor CLI provides a Developer Friendly way to interact with KubeArmor Telemetry. You can stream KubeArmor telemetry independently of kArmor CLI tool and integrate it with your chosen SIEM (Security Information and Event Management) solutions. [Here's a guide](https://github.com/kubearmor/kubearmor-relay-server/blob/main/README.md#streaming-kubearmor-telemetry-to-external-siem-tools) on how to achieve this integration. This guide assumes you have kArmor CLI to access KubeArmor Telemetry but you can view it on your SIEM tool once integrated.
 
+## KubeArmor EKS-plugin 
+
+> Amazon Elastic Kubernetes Service (Amazon EKS) is a managed Kubernetes service provided by Amazon Web Services (AWS) which simplifies the process of running Kubernetes clusters by handling the underlying infrastructure and operational aspects. KubeArmor’s integration with EKS can provide a user with a tool that can enforce security policies for them at the pod level as well as at the node level. Users can restrict all events related to network, process, and file using these policies.
```
Comment (daemon1024) [inline getting-started/deployment_guide.md]: I don't think we need to explain EKS and KubeArmor here. It's a KubeArmor deployment guide so people already know about Kubernetes and Kubearmor here. Linking to the description of EKS in a hyperlink would be enough. WDYT?

**Review by daemon1024 — CHANGES_REQUESTED** [review]

File: getting-started/deployment_guide.md (hunk 1 above)
Comment (Chirag-CG) [reply getting-started/deployment_guide.md]: Created a collapsible section for Eks-plugin installation under install .

File: getting-started/deployment_guide.md (hunk 2 above)
Comment (Chirag-CG) [reply getting-started/deployment_guide.md]: Removed unnecessary information and hyperlinked EKS-plugin official documentation.

File: getting-started/deployment_guide.md [hunk 3]
```diff
@@ -14,6 +14,45 @@ kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/pkg/
...
+<details>
+  <summary><h4>EKS-Plugin installation</h4></summary>
+  
+Here, you will understand how to install KubeArmor on your EKS cluster using [Amazon EKS add-on](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html). 
+
+### From AWS Console
+
+Once you've subscribed to Kubearmor on AWS Marketplace and completed the on-screen setup, you'll be directed to the Amazon EKS console. To begin in the Amazon EKS console, navigate to your EKS clusters and click on the "Add-ons" tab. From there, select "Get more add-ons" to locate the KubeArmor EKS add-ons in the cluster settings of your existing EKS clusters. You can use the search bar to find "KubeArmor" and then follow the on-screen instructions to enable the Kubearmor add-on for your Amazon EKS cluster.
+
+### From AWS-CLI
```
Comment (daemon1024) [inline getting-started/deployment_guide.md]:
```suggestion
**From AWS Console**

Once you've subscribed to Kubearmor on AWS Marketplace and completed the on-screen setup, you'll be directed to the Amazon EKS console. To begin in the Amazon EKS console, navigate to your EKS clusters and click on the "Add-ons" tab. From there, select "Get more add-ons" to locate the KubeArmor EKS add-ons in the cluster settings of your existing EKS clusters. You can use the search bar to find "KubeArmor" and then follow the on-screen instructions to enable the Kubearmor add-on for your Amazon EKS cluster.

**From AWS-CLI**
```

Main heading is h4, so let's move these headings to h4 or bold (h5 doesn't exist)

File: getting-started/deployment_guide.md [hunk 4]
```diff
@@ -14,6 +14,45 @@ kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/pkg/
 
 You can find more details about helm related values and configurations [here](https://github.com/kubearmor/KubeArmor/tree/main/deployments/helm/KubeArmorOperator).
 
+<details>
+  <summary><h4>EKS-Plugin installation</h4></summary>
+  
+Here, you will understand how to install KubeArmor on your EKS cluster using [Amazon EKS add-on](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html).
```
Comment (daemon1024) [inline getting-started/deployment_guide.md]:
```suggestion
If you are using AWS EKS, You can use [Amazon EKS add-on](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html) to install KubeArmor in your AWS EKS Cluster. 
```

Please improvise as needed

**Review by daemon1024 — CHANGES_REQUESTED** [review]

File: getting-started/deployment_guide.md (hunk 3 above)
Comment (Chirag-CG) [reply getting-started/deployment_guide.md]: Resolved

File: getting-started/deployment_guide.md (hunk 4 above)
Comment (Chirag-CG) [reply getting-started/deployment_guide.md]: Resolved

**Comment by nyrahul** [comment]
Let's not have provider specific installation point on the main getting started guide. We can have a link and then describe.

**Comment by Chirag-CG** [comment]
> Let's not have provider specific installation point on the main getting started guide. We can have a link and then describe.

So do we publish this help.accuknox.com and link it here ?

**Comment by daemon1024** [comment]
We don't want to link it in the getting started guide itself.

Let's link it in the support matrix where support for AWS EKS is mentioned.

**Comment by nyrahul** [comment]
THe information is pushed [here](https://help.accuknox.com/integrations/ka-eks-add-on/). Closing the PR.

**Pull request closed without merging**